GITHUB-TEAMS.md

```diff
@@ -0,0 +1,18 @@
+In order to ease notifying folks with a common expertise or interest, WHATWG uses GitHub teams. WHATWG uses teams in the following manner:
+
+* Anyone can join any team they desire. To join just let someone know or [file an issue](https://github.com/whatwg/meta/issues/new).
+* Everyone tries to ensure the appropriate teams are pinged on relevant issues/pull requests.
+* Teams have no formal authority, though it's fine to add them as contributors to repositories for ease of collaboration. For all repositories we care about protecting master primarily. The bar for write access to other branches is much much lower.
+* Teams have no responsibility either. If you ping a team and get no feedback, moving things forward responsibly is still on you.
+* Creating new teams is as simple as [filing an issue](https://github.com/whatwg/meta/issues/new). Review should take care it is not too much of a niche or overlap with existing teams.
+
+At the time of writing, the following teams exist:
```

Comment:
Can we just link to https://github.com/orgs/whatwg/teams ? This seems like it would get out of date easily... I wonder if we can link to https://github.com/orgs/whatwg/teams ? I'm not sure whether that's visible to everyone though; it's not if you're logged out...

Reply:
That's why I recreated the list.